docs/user-guide/workflow.ipynb

```diff
@@ -87,8 +87,13 @@
    "outputs": [],
    "source": [
     "workflow = pol.he3.He3CellWorkflow()\n",
-    "results = (pol.PolarizationCorrectedSampleData,)\n",
-    "workflow.visualize(results, graph_attr={'rankdir': 'LR'})"
+    "results = (\n",
+    "    pol.PolarizationCorrectedData[pol.Up, pol.Up],\n",
+    "    pol.PolarizationCorrectedData[pol.Up, pol.Down],\n",
+    "    pol.PolarizationCorrectedData[pol.Down, pol.Up],\n",
+    "    pol.PolarizationCorrectedData[pol.Down, pol.Down],\n",
+    ")\n",
+    "workflow.visualize(results, graph_attr={\"rankdir\": \"LR\"})"
    ]
```

Comment:
- In this image, what is the difference between the `He3CellTransmissionFractionHasPolarizedIncomingBeam` and the `He3CellTransmissionFraction[Polarizer/Analyzer]`? 
- Is the first one just an input to decide that we use an incoming polarized beam or not?
- This does not yet include to put in the measurements from both incoming polarization states (plus/minus) and make a combined fit, right?

Reply:
> Is the first one just an input to decide that we use an incoming polarized beam or not?

Yes, and I expect this mechanism to change when we implement #58

> This does not yet include to put in the measurements from both incoming polarization states (plus/minus) and make a combined fit, right?

Right, see #58.